corehq/warehouse/transforms/sql/user_location_dim.sql

```diff
@@ -0,0 +1,66 @@
+--- Delete all records that appear in the user staging table
+DELETE FROM {{ user_location_dim }} as ul_dim
+USING {{ user_staging }} as user_staging
+INNER_JOIN {{ user_dim }} as ud
+ON ud.user_id = user_staging.user_id
+
+WHERE ul_dim.user_dim = ud.id;
+
+
+--- Repopulate the UserLocationDim
+INSERT INTO {{ user_location_dim }} (
+       domain,
+       user_dim_id,
+       location_dim_id,
+       dim_last_modified,
+       dim_created_on,
+       deleted,
+       batch_id
+)
+--- webusers
+SELECT
+	domain,
+	user_dim_id,
+	location_dim_id,
+	now(),
+	now(),
+	false,
+	{{ batch_id }}
+FROM
+(
+	SELECT
+		user_dim.id as user_dim_id,
+		json_array_elements(json_array_elements(user_staging.domain_memberships::JSON) -> 'assigned_location_ids') as location_id,
```

Comment:
could there be more than 1 location id from this?

Reply:
Yeah, from my understanding (and some testing), this will expand to one row per location id